Formula/gmt.rb

```diff
@@ -14,8 +14,10 @@ class Gmt < Formula
   end
 
   depends_on "cmake" => :build
```

Comment:
GMT 6.0.0 also depends on the libcurl library to download remote files.

Reply:
Does it mean that it will automatically download "gshhg" and "dcw" files at compile time?

Reply:
> Does it mean that it will automatically download "gshhg" and "dcw" files at compile time?

No. Since GMT 5.4, GMT provides earth relief data with different resolutions. So, if you run 
```
gmt grdinfo @earth_relief_06m
```
GMT will autoamatically download the earth relief grid data (6 m resolution) from GMT's server and save it to your local cache (~/.gmt/server).

Reply:
Ok. Anyway libcurl is part of standard macOS installation, i.e. `/usr/lib/libcurl.dylib`. Should we do something particular here?

Reply:
> Ok. Anyway libcurl is part of standard macOS installation, i.e. `/usr/lib/libcurl.dylib`. Should we do something particular here?

homebrew also provides its own `curl`. I don't know the difference between them.